glue_connection.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_glue_connection" "this" {
+  count = var.enable_glue_connection ? 1 : 0
+  name  = "${var.name}-${random_string.identifier.result}"
+  tags  = merge(local.tags, var.tags)
+  connection_properties = {
+    JDBC_CONNECTION_URL = "jdbc:mysql://${aws_rds_cluster.this.endpoint}/${aws_rds_cluster.this.database_name}"
```

Comment:
I think port needs to be here too. 
`"jdbc:mysql://${aws_rds_cluster.this.endpoint}:${port}/${aws_rds_cluster.this.database_name}"`

Reply:
added port